website/src/categories/components/component-table.njk

```diff
@@ -35,7 +35,7 @@
                   </thead>
                   <tbody>
                   {%- for member in declaration.members -%}
-                    {%- if member.kind == 'field' -%}
+                    {%- if member.kind == 'field' and not member.static -%}
```

Comment:
I'm not sure it's always right? For example in the breadcrumbs we want to show that part:
<img width="1057" alt="image" src="https://github.com/user-attachments/assets/49fdc20e-6dcc-4021-8ac2-1c00d134fce5">

Maybe we need to take a look at components and when the static should not be really shown we need to add `@ignore` there? :)

Reply:
those properties are still visible on the website. Right now they are displayed twice: 
<img width="1080" alt="image" src="https://github.com/user-attachments/assets/de8783b1-0ca0-4c23-9f55-69e48fd89d16">